internal/command/providers_lock.go

```diff
@@ -243,9 +245,17 @@ func (c *ProvidersLockCommand) Run(args []string) int {
 		}
 		ctx := evts.OnContext(ctx)
 
+		// We can not use c.providerGlobalCacheDir() as we install the provider into a temp dir
```

Comment:
do we need to remove this comment as well?

Reply:
Yeah, I think it's more confusing than helpful